modin/pandas/series.py

```diff
@@ -911,12 +905,6 @@ def dot(self, other):  # noqa: PR01, RT01, D200
             query_compiler=self._query_compiler.dot(other, squeeze_self=True)
         )
 
-    def drop_duplicates(self, keep="first", inplace=False):  # noqa: PR01, RT01, D200
```

Comment:
This has extra params `kwargs`: https://github.com/modin-project/modin/runs/7982177596?check_suite_focus=true